ui/app/AppLayouts/Profile/views/RPCStatsModal.qml

```diff
@@ -0,0 +1,163 @@
+import QtQuick 2.15
+import QtQml 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import shared.controls 1.0
+import utils 1.0
+
+import SortFilterProxyModel 0.2
+
+import "../stores"
+
+
+StatusDialog {
+    id: root
+
+    implicitHeight: 610
+
+    property WalletStore walletStore
+
+    QtObject {
+        id: d
+
+        property int totalCalls
+        property int totalFilteredCalls: countAggregator.value
+
+        function updateModel() {
+            sourceModel.clear()
+
+            const jsonStatsRaw = root.walletStore.getRpcStats()
+            const jsonStats = JSON.parse(jsonStatsRaw)
+            if (jsonStats.result && jsonStats.result.methods) {
+                const methods = jsonStats.result.methods
+                d.totalCalls = jsonStats.result.total
+
+                for (const method in methods) {
+                    sourceModel.append({ method: method, count: methods[method] })
+                }
+            }
+        }
+
+        function resetStats() {
+            root.walletStore.resetRpcStats()
+            updateModel()
+        }
+    }
+
+    ColumnLayout {
+        id: contentColumn
+
+        anchors.fill: parent
+
+        SearchBox {
+            id: searchBox
+
+            Layout.fillWidth: true
+            Layout.bottomMargin: 16
+        }
+
+        StatusListView {
+            id: resultsListView
+
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            spacing: 2
+
+            ListModel {
+                id: sourceModel
+            }
+
+            header: StatusListItem {
+                title: qsTr("Total")
+                statusListItemTitle.customColor: Theme.palette.directColor1
+                statusListItemLabel.customColor: Theme.palette.directColor1
+                label: d.totalFilteredCalls + " of " + d.totalCalls
```

Comment:
Kinda strange you mark something as translatable (`title`) above something not :)
```suggestion
                label: qsTr("%1 of %2).arg(d.totalFilteredCalls)..arg(d.totalCalls)
```